src/librfu_rfu.c

```diff
@@ -1400,7 +1400,11 @@ static u16 rfu_STC_setSendData_org(u8 ni_or_uni, u8 bmSendSlot, u8 subFrameSize,
 {
     u8 bm_slot_id, sendSlotFlag;
     u8 frameSize;
+#ifdef UBFIX
+    u8 *llFrameSize_p = NULL;
+#else
     u8 *llFrameSize_p;
+#endif
```

Comment:
This doesn't actually fix anything (aside from silencing an uninitialized warning). In the case where this would be used it'd still be undefined behavior, because you're dereferencing a null pointer.

Reply:
I knew I forgot to copy something from the PR :(
I'll apply the null pointer dereferencing fixes as well.